pyiode/objects/scalar.pyx

```diff
@@ -22,6 +23,8 @@ cdef class Scalar:
     Parameters
     ----------
     value: float
+        # GDM> it is very odd to have a default value here. I guess that's because iode the GUI has one, but in
```

Comment:
In the estimation process, scalars with default *value* and *relax* are created if not found in the global Scalars workspace. So it was legitimate to have a C++ constructor with a default value for *value*. 
In the GUI also, when you open the box to add a new scalar, the *value* field already contains the value 0.9 by default.

But I agree with you that the users should be forced to specify a value for *value* when creating a new scalar in Python

Reply:
Default value for *value* removed in PR #581